source/s3_endpoint_resolver/aws_s3_endpoint_resolver_partition.c

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates.
+ * All Rights Reserved. SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/s3/s3_endpoint_resolver/s3_endpoint_resolver.h>
+
+/**
+ * This file is generated using a Python script.
```

Comment:
trivial: may as well mention the script by name/path

Reply:
Thanks, updated.